src/format.rs

```diff
@@ -313,7 +313,7 @@ impl Documentable for &UntypedPattern {
         match self {
             Pattern::Int { value, .. } => value.clone().to_doc(),
 
-            Pattern::Float { value, .. } => value.to_doc(),
+            Pattern::Float { value, .. } => value.clone().to_doc(),
```

Comment:
Gleam allows floats to omit the numbers after the `.`. In the Erlang we want the number to always have a number after the `.` so here we can check to see if the `value` string ends with `.` and add a 0 to the end if it does: `format!("{}0", value).to_doc()`